src/simple_cache_client.rs

```diff
@@ -1991,13 +1964,15 @@ impl SimpleCacheClient {
         set_name: impl IntoBytes,
         element_name: impl IntoBytes,
     ) -> MomentoResult<Option<u64>> {
+        use momento_protos::cache_client::sorted_set_get_rank_request::Order::Ascending;
         use momento_protos::cache_client::sorted_set_get_rank_response::Rank;
 
         let request = self.prep_request(
             cache_name,
             SortedSetGetRankRequest {
                 set_name: set_name.into_bytes(),
-                element_name: element_name.into_bytes(),
+                value: element_name.into_bytes(),
+                order: Ascending.into(),
```

Comment:
This is not changing the behavior of this method at all. The `Ascending` enum has a value of 0, which is the default value that the proto would have. Since its now required by the `SortedSetGetRankRequest`, I am just passing in what is the default value. We need to come back into this sdk and rework the sorted set methods.

```
message _SortedSetGetRankRequest {
  enum Order {
    ASCENDING = 0;
    DESCENDING = 1;
  }
```